### PR TITLE
21 user creation

### PR DESCRIPTION
**Pull request by MrBennySolomon**

### CODE REVIEW

File: server/server.js [hunk 1]
```diff
@@ -38,7 +40,12 @@ wss.on("connection", (ws) => {
   };
 });
 
-const PORT = process.env.PORT || 5005;
+app.post("/user-data", async (req, res) => {
+  const userData = req.body;
+  usersMap.set(userData.name, userData);
```
Comment (yishain11) [inline server/server.js:45]: where is the mongodb collection creation?
storgae?
read the issue description

**Review by yishain11 — CHANGES_REQUESTED** [review]

File: server/api/translation/openAI.js [hunk 2]
```diff
@@ -1,46 +1,46 @@
...
-    model: "gpt-3.5-turbo",
-    messages: [{ role: "user", content: prompt }],
+const init = () => {
+  const config = new Configuration({
+    apiKey: process.env.OPEN_AI_API_KEY,
   });
+  return new OpenAIApi(config);
 };
 
-export const checkProfanity = async (msg) => {
```
Comment (yishain11) [inline server/api/translation/openAI.js:18]: why did you delete this?

File: server/api/translation/openAI.js [hunk 3]
```diff
@@ -1,46 +1,46 @@
...
-  const openai = new OpenAIApi(
-    new Configuration({ apiKey: process.env.OPEN_AI_API_KEY })
-  );
-  const isProfanity = await sendPromptToOpenAi(
-    openai,
-    `is the text profanity? answer only "true" or "false" : ${msg}`
-  );
+const openAiInit = async (openai, prompt) => {
```
Comment (yishain11) [inline server/api/translation/openAI.js:17]: why did you add this? we are working with chat, not with text completion. fix

**Review by yishain11 — CHANGES_REQUESTED** [review]
please read the issue:
where is the function to create users schema at server start - if not exists?

File: server/server.js [hunk 4]
```diff
@@ -38,7 +43,11 @@ wss.on("connection", (ws) => {
   };
 });
 
-const PORT = process.env.PORT || 5005;
+app.post("/user-data", async (req, res) => {
```
[the remark this answers is not in the document]
Comment (MrBennySolomon) [reply server/server.js]: it's ok, it's in the user.routes.js file

File: server/api/translation/openAI.js (hunk 2 above)
Comment (MrBennySolomon) [reply server/api/translation/openAI.js:18]: it's ok , i changed it last week, and this issue is before 2 weeks

File: server/api/translation/openAI.js (hunk 3 above)
Comment (MrBennySolomon) [reply server/api/translation/openAI.js:17]: it's ok , i changed it last week, and this issue is before 2 weeks

File: server/server.js [hunk 5]
```diff
@@ -16,13 +19,15 @@ const wss = new WebSocketServer({ port: process.env.WEB_SOCKET_PORT });
...
 app.use(express.json());
+app.use(cors());
 
 connectDB();
 
 const rooms = {};
-
 const clients = [];
 
+connectDB();
```
Comment (yishain11) [inline server/server.js:29]: remove. duplicate to line 24

**Review by yishain11 — CHANGES_REQUESTED** [review]

**Review by yishain11 — CHANGES_REQUESTED** [review]
please fix:
1. import errors in app.jsx
2. the pages you changed in relation to their state in main
